### PR TITLE
Add default cloud users to cloud-init

### PR DESCRIPTION
**Pull request by vatsalparekh**
Signed-off-by: Vatsal Parekh <vparekh@redhat.com>

**What this PR does / why we need it**:
Adding the default user to the cloud-init gives an easy way to check it

**Which issue(s) this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close the issue(s) when PR gets merged)*:
Fixes # [BZ1879483](https://bugzilla.redhat.com/show_bug.cgi?id=1879483)

**Special notes for your reviewer**:

**Release note**:

```release-note
Add default cloud users to cloud-init
```

### CODE REVIEW

File: generate-templates.yaml [hunk 1]
```diff
@@ -170,6 +176,7 @@
       icon: fedora
       oslabels: "{{ fedora_labels }}"
       osinfoname: "{{ oslabels[0] }}"
+      cloudusername: fedora
```
Comment (ksimon1) [inline generate-templates.yaml:179]: Why RHELs have `cloudusername: cloud-user` but others have `cloudusername` their name?

Comment (vatsalparekh) [reply generate-templates.yaml:179]: `cloud-user` is the default user name for RHEL
The purpose here is to specify the default usernames to better UX in the console

**Review by omeryahud — APPROVED** [review]

**Pull request merged**